apps/native-component-list/screens/FileSystemScreen.js

```diff
@@ -22,6 +22,12 @@ export default class FileSystemScreen extends React.Component {
   };
 
   _download = async () => {
+    const url = 'http://ipv4.download.thinkbroadband.com/256KB.zip';
```

Comment:
Do you think it would make sense to create some small server like this one: https://github.com/expo/authenticated-static-files, deploy on now.sh and use in these tests and in `test-suite`? Maybe it would make sense to just extend this ^ app?

Reply:
I think it makes sense. Never used now before, but it looks fantastic for that case.